examples/htool_console.c

```diff
@@ -322,9 +320,9 @@ int htool_console_snapshot(struct libhoth_device *dev) {
       fprintf(stderr, "EC_CMD_CONSOLE_READ status: %d\n", status);
       return status;
     }
-    strncat(ret, buf, sizeof(buf));
+    printf("%s", buf);
```

Comment:
If haven doesn't null-terminate the snapshot, this could print out garbage from the stack.

I think you want something like

```
fwrite(buf, strnlen(buf, sizeof(buf)), 1, stdout);
```

Reply:
Ack. Fixed